src/main/java/me/totalfreedom/totalfreedommod/command/Command_whitelist.java

```diff
@@ -198,10 +197,21 @@ else if (args[0].equals("remove"))
     public List<String> getWhitelistedNames()
     {
         List<String> names = new ArrayList<>();
-        for (Object name : plugin.si.getWhitelisted())
+        for (OfflinePlayer player : server.getWhitelistedPlayers())
         {
-            names.add(String.valueOf(name));
+            names.add(player.getName());
         }
         return names;
     }
+
+    public int purge()
+    {
+        int removed = 0;
+        for (OfflinePlayer player : server.getWhitelistedPlayers())
```

Comment:
Does this account for people who are online as well?

Reply:
Yes.

Reply:
Then everything else looks fine to me.